.gitignore

```diff
@@ -1,3 +1,6 @@
+# Autogenerated version file
+/adb_shell/version.py
```

Comment:
If a user doesn't have this file, then they won't be able to test the package locally. Is there a command to generate `version.py`?

Reply:
`python3 -m build -nwx` It builds a wheel, and `setuptools_scm` generates the file in process.